README.md

```diff
@@ -1 +1,46 @@
-# common-pipeline-templates
\ No newline at end of file
+# PINS Pipeline Templates
```

Comment:
Maybe add a section for installing tools locally, similar to:
https://github.com/Planning-Inspectorate/infrastructure-environments#installation--setup

Reply:
This is covered by the Setup section no? It's just node and npm that are needed.